main.js

```diff
@@ -0,0 +1,22 @@
+const STICKERS_URL = 'http://localhost:3000/api/v1/stickers'
```

Comment:
What are these stickers, I don't see that this url was created anywhere? Also, a localhost url will not work once on production